charts/trillian/templates/trillian-log-server/deployment.yaml

```diff
@@ -47,6 +47,19 @@ spec:
       priorityClassName: "{{ .Values.logServer.priorityClassName }}"
 {{- end }}
       containers:
+        {{- if .Values.mysql.gcp.enabled }}
```

Comment:
Should we include a new value to support user defined arbitrary containers?

Reply:
might be helpful, idk what other differences other cloud providers would require here.